Dispatcher/Controller/SplashViewController.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+class SplashViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + 3.0) {
+            let storyboard = UIStoryboard(name: "Main", bundle: nil)
+            let tabBarController = storyboard.instantiateViewController(withIdentifier: "TabBarController") as! UITabBarController
```

Comment:
please notice you are using what is called a "crash operator" which is "!"
meaning the application will crash on purpose when "something isn't there"

if the view controller wont be found in the storyboard the application will crash,
in this case it is fine to use since without this we cant run the application anyway.

but bear in mind that using this operator is bad practice and should be used as little as possible
you can keep it the way it is here